db/migrate/20231026032737_create_posts.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePosts < ActiveRecord::Migration[7.1]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.text :text
+      t.integer :commentsCounter
+      t.integer :likesCounter
+      t.references :users,foreign_key:true
+      t.timestamps
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/creating_data_model_project_v1_2.md#:~:text=should%20be%20included.-,Pay%20attention%20that%20in%20posts%20table%20the%20column%20holding%20the%20foreign%20key%20to%20the%20users%20table%20should%20be%20named%20author_id,-All%20columns%20that), The foreign key between the users and posts tables is `author_id`, not `user_id` as you did here. Please fix that.

[Hint] It will be like that instead:
```ruby
t.references :author, foreign_key: { to_table: 'users' }
```